ckanext/datarequests/controllers/ui_controller.py

```diff
@@ -263,7 +263,7 @@ def delete(self, id):
             log.warn(e)
             tk.abort(404, tk._('Data Request %s not found') % id)
         except tk.NotAuthorized as e:
-            log.warn(e) 
+            log.warn(e)
```

Comment:
how big is the exceptions, we might want a short string instead to be logged if its a full stack trace.

Reply:
There is no stack trace in a NotAuthorized exception(https://github.com/ckan/ckan/blob/master/ckan/logic/__init__.py#L55), just a message (https://github.com/ckan/ckan/blob/master/ckan/logic/__init__.py#L31) that can be defined in the auth function. But after checking the auth functions from the original programmer, he did not include any messages so the log which comes back is only `ckan_1 | 17:07:40,863 WARNI [ckanext.datarequests.controllers.ui_controller]`